exercises/two-bucket/README.md

```diff
@@ -0,0 +1,48 @@
+# Two Bucket
+
+Given two buckets of different size, demonstrate how to measure an exact number of liters by strategically transferring liters of fluid between the buckets.
+
+Since this mathematical problem is fairly subject to interpretation / individual approach, the tests have been written specifically to expect one overarching solution.
+
+To help, the tests provide you with which bucket to fill first. That means, when starting with the larger bucket full, you are NOT allowed at any point to have the smaller bucket full and the larger bucket empty (aka, the opposite starting point); that would defeat the purpose of comparing both approaches!
+
+Your program will take as input:
+- the size of bucket one
+- the size of bucket two
+- the desired number of liters to reach
+- which bucket to fill first, either bucket one or bucket two
+
+Your program should determine:
+- the total number of "moves" it should take to reach the desired number of liters, including the first fill
+- which bucket should end up with the desired number of liters (let's say this is bucket A) - either bucket one or bucket two
+- how many liters are left in the other bucket (bucket B)
+
+Note: any time a change is made to either or both buckets counts as one (1) move.
+
+Example:
+Bucket one can hold up to 7 liters, and bucket two can hold up to 11 liters. Let's say bucket one, at a given step, is holding 7 liters, and bucket two is holding 8 liters (7,8). If you empty bucket one and make no change to bucket two, leaving you with 0 liters and 8 liters respectively (0,8), that counts as one "move". Instead, if you had poured from bucket one into bucket two until bucket two was full, leaving you with 4 liters in bucket one and 11 liters in bucket two (4,11), that would count as only one "move" as well.
+
+To conclude, the only valid moves are:
+- pouring from one bucket to another
+- emptying one bucket and doing nothing to the other
+- filling one bucket and doing nothing to the other
+
+Written with <3 at [Fullstack Academy](http://www.fullstackacademy.com/) by [Lindsay](http://lindsaylevine.com).
```

Comment:
Looks like this link is dead...

Actually, looking at the [metadata](https://github.com/exercism/problem-specifications/blob/master/exercises/two-bucket/metadata.yml), this README might be out of date.

Make sure you have the most recent version of `problem-specifications` before generating the README! From your problem-specifications clone:

```bash
git checkout master
git remote add upstream https://github.com/exercism/problem-specifications.git
git fetch upstream
git merge upstream/master
```

Then regenerate it 😄

Reply:
I followed your instructions and regenerated the README, but I think it is the same as before.

Reply:
@jssander and @sjwarner-bp the README here is up to date, [the problem-specification description](https://github.com/exercism/problem-specifications/blob/master/exercises/two-bucket/description.md) has the broken link

Reply:
The README where?

Reply:
@jssander I was referring to the one you generated. It is up to date with the canonical data, it's just the canonical data had the broken link in the description for the exercise. I've submitted a pull request over there to resolve the issue.

Reply:
Ah OK, I must have gotten confused because of the Lindsay link! I think I submitted a PR (maybe for a different exercise) with the same broken link a while ago 🙂

In that case once the indentation is fixed I am happy to merge it and will regenerate the README.md myself when the changes are merged to problem specifications!